examples/all-clusters-minimal-app/all-clusters-common/all-clusters-minimal-app.matter

```diff
@@ -4618,17 +4724,57 @@ endpoint 1 {
   }
 }
 endpoint 2 {
+  device type ma_powersource = 17, version 1;
   device type ma_onofflight = 256, version 1;
 
 
+  server cluster Identify {
```

Comment:
Again, this is probably broken in practice (on endpoint 2 here) and needs C++ changes to make it actually work.  Followup, please.

Reply:
https://github.com/project-chip/connectedhomeip/issues/28658